resources/js/Pages/Admin/Rooms/UpdateRoomForm.vue

```diff
@@ -69,6 +69,19 @@
                 />
             </div>
 
+             <div class="m-6">
+                <jet-label for="room_type" value="room_type" />
+                <select v-model="form.status" class="mt-1 block w-full" name="room_type" id="room_typeUpdate">
+                    <option :value="form.status" selected="selected">{{form.status}}</option>
+                    <option v-if="form.status != 'available'" :value="'available'">available</option> 
+                    <option v-else :value="'unavailable'">anavailable</option>
```

Comment:
this looks like this was a copy paste of status. make sure you add the proper room type options here